lib/orderbook/OrderBook.ts

```diff
@@ -121,14 +127,17 @@ class OrderBook extends EventEmitter {
     return this.addOwnOrder({ ...order, price }, true);
   }
 
-  public removeOwnOrderByLocalId = (pairId: string, localId: string): boolean => {
+  public removeOwnOrderByLocalId = (pairId: string, localId: string): { removed: boolean, globalId: string } => {
     const id = this.localIdMap[localId];
 
     if (id === undefined) {
-      return false;
+      return { removed: false, globalId: '' };
```

Comment:
globalId can be undefined rather than a empty string.

Reply:
I think we should change this to `undefined` as well.